packages/synchro-charts/cypress/integration/charts/line-chart/draggable-annotations.spec.ts

```diff
@@ -240,10 +240,10 @@ it('draggable annotation rescales axis in negative direction', () => {
   cy.get('g.axis.y-axis > g.tick').should('have.length', 10);
   cy.get('g.axis.y-axis > g.tick > text')
     .eq(9)
-    .should('have.text', '5,000');
+    .should('have.text', '5k');
```

Comment:
is the number of significant digits based on the overall y-axis range?

Reply:
yep! if the values are greater than 10k apart it wont add in the significant digits

examples: `0 -> 6000`
<img width="300" alt="image" src="https://user-images.githubusercontent.com/28601414/199255312-338af1fd-f044-410a-80b8-0bee099bcc79.png">
`7000 -> 12000`
<img width="300" alt="image" src="https://user-images.githubusercontent.com/28601414/199256202-4792f78b-84da-4c13-b99c-8c997c3fa35c.png">
`7000 -> 16500`
<img width="300" alt="image" src="https://user-images.githubusercontent.com/28601414/199256461-f9f46881-4f19-41ab-9446-2cfc02f65c49.png">
`7000 -> 17000`
<img width="300" alt="image" src="https://user-images.githubusercontent.com/28601414/199256306-0b2d882c-eb39-4ce3-8210-9da11927e357.png">